.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
   - '10'
-  - '12'
+  - '12.13.0'
```

Comment:
the `/.bin/jest` module loading error  is only on node 12.13.1...

Reply:
Nice catch :+1: 

Should we add a comment and/or a redmine issue to track it? Assuming we would want to bring it back to 12 once they fix the issue.

Reply:
I added a comment for it